web/templates/note/edit.html.eex

```diff
@@ -4,4 +4,6 @@
   conn: @conn,
   changeset: @changeset,
   action: note_path(@conn, :update, @note),
-  submit_text: "Update Note" %>
+  submit_text: "Update Note",
+  cancel_link:  link "Cancel", 
```

Comment:
I would use parenthesis for this `link`. I think we're depending a little too much here on the parser having the right precedence for whitespace. I think if you wanted to add another key/value pair to the render call at the end, it would end up being associated with the `link`, not the `render` call. Better to be clear.